service/javadsl/api/src/main/scala/com/lightbend/lagom/internal/javadsl/api/ServiceReader.scala

```diff
@@ -221,8 +221,18 @@ object ServiceReader {
   }
 
   private def constructServiceCallHolder[Request, Response](serviceCallResolver: ServiceCallResolver, method: Method): ServiceCallHolder = {
+
     val serializers = method.getGenericParameterTypes.toSeq.map { arg =>
-      serviceCallResolver.resolvePathParamSerializer(new UnresolvedTypePathParamSerializer[AnyRef], arg)
+      try {
+        serviceCallResolver.resolvePathParamSerializer(new UnresolvedTypePathParamSerializer[AnyRef], arg)
+      } catch {
+        case ex: IllegalArgumentException =>
+          throw new RuntimeException(s"Error encountered while resolving the ${method.getDeclaringClass + "." + method.getName}" +
+            "service call: No path parameter serializer was found for the com.example.Foo path parameter. This can be fixed " +
```

Comment:
Is `com.example.Foo` an oversight?

Reply:
yes, it seems I missed it. I'll fix it asap.

Reply:
@ignasi35 I  updated the error message with correct value